locksmith/__tests__/operations/userOperations.test.ts

```diff
@@ -0,0 +1,86 @@
+import UserOperations from '../../src/operations/userOperations'
+
+const models = require('../../src/models')
+let User: any = models.User
```

Comment:
I think we want to avoid `any`, right @cnasc ?

Reply:
At the moment, there are not type definitions defined for these generated objects. This can be done, but I think that it can be done after this lands

Reply:
I agree with @akeem on this one. `any` is undesirable, and in many cases it's trivially easy to get rid of it. But if in this case there's no provided typedef, it's probably more important to take it down as a TODO so we can keep moving forward.